src/main/kotlin/io/easybreezy/infrastructure/event/project/project/CategoryAdded.kt

```diff
@@ -0,0 +1,22 @@
+package io.easybreezy.infrastructure.event.project.project
+
+import io.easybreezy.infrastructure.event.Event
+import io.easybreezy.infrastructure.serialization.LocalDateTimeSerializer
+import io.easybreezy.infrastructure.serialization.UUIDSerializer
+import kotlinx.serialization.Serializable
+import java.time.LocalDateTime
+import java.util.UUID
+
+@Serializable
+data class CategoryAdded(
+    @Serializable(with = UUIDSerializer::class) val project: UUID,
+    @Serializable(with = UUIDSerializer::class) val category: UUID,
+    val name: String,
+    @Serializable(with = UUIDSerializer::class) val parent: UUID?,
```

Comment:
You can set a file seriliation rules `@file:UseSerializers(LocalDateSerializer::class)`. For example file `io.easybreezy.integration.gitlab.webhook.action.pipeline`